engine/flink/util/src/main/scala/pl/touk/nussknacker/engine/flink/util/keyed.scala

```diff
@@ -53,8 +54,7 @@ object keyed {
 
   }
 
-  /* TODO, FIXME: Errors on interpret should be handled with FlinkEspExceptionHandler; see KeyedRecordFlatMapper */
-  abstract class BaseKeyedValueMapper[OutputKey <: AnyRef: TypeTag, OutputValue <:AnyRef: TypeTag] extends RichMapFunction[Context, ValueWithContext[KeyedValue[OutputKey, OutputValue]]] with LazyParameterInterpreterFunction {
+  abstract class BaseKeyedValueMapper[OutputKey <: AnyRef: TypeTag, OutputValue <:AnyRef: TypeTag] extends RichFlatMapFunction[Context, ValueWithContext[KeyedValue[OutputKey, OutputValue]]] with LazyParameterInterpreterFunction {
```

Comment:
Shouldn't we made some changes also in `KeyedRecordFlatMapper` after this change so it can look like other `LazyParameterInterpreterFunction`s ?

Reply:
true :+1: